src/commands/bot-credentials/list.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as BotCredentialsAPI from '../../mix/api/bot-credentials'
+import * as MixFlags from '../../utils/flags'
+import {BotCredentialsParams, MixClient, MixResponse, MixResult} from '../../mix/types'
+import {DomainOption} from '../../utils/validations'
+import MixCommand from '../../utils/base/mix-command'
+
+const debug = makeDebug.debug('mix:commands:bot-credentials:list')
+
+export default class BotCredentialsList extends MixCommand {
+  static description = `list credentials for a bot
+  
+Use this command to list the credentials for a bot.
+This lets you retrieve the bot ID that is required in other
```

Comment:
Replace the paragraph here with:

Use this command to list the credentials for a bot.
This lets you retrieve the runtime application ID that is required
in other commands.